citeproc-java/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
     dependencies {
-        classpath 'org.apache.commons:commons-lang3:3.4'
+        classpath 'org.apache.commons:commons-lang3:3.9'
     }
 }
 
 plugins {
-    id "de.undercouch.download" version "3.4.3"
-    id "nebula.optional-base" version "3.0.3"
+    id 'de.undercouch.download' version '4.0.0'
```

Comment:
Version 4.0.1 is already available, which contains an important bugfix

Reply:
:+1: